content/en/user-guide/tags/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Tags"
+linkTitle: "Tags"
+weight: 15
+description: Get started with Tags in LocalStack for Snowflake
+---
+
+## Introduction
+
+Snowflake tags allow you to categorize and manage Snowflake objects by associating custom metadata with them. These tags support governance, cost tracking, and data lineage by enabling organizations to label resources with business-relevant information.
+
+The Snowflake emulator supports tags, allowing you to apply these tags to the local Snowflake tables, views, and databases using the same commands and syntax as the Snowflake service. The following operations are supported:
+
+-   [CREATE TAG](https://docs.snowflake.com/en/sql-reference/sql/create-tag)
```

Comment:
It would be great to have consistent formatting in storage integration PR I've seen `CREATE TAG` (with backticks).